app.py

```diff
@@ -129,12 +127,36 @@ def _register_indexes(graph: GraphDatabase, index_base_url: str, dry_run: bool =
     return index_urls
 
 
-def _do_schedule_core_solver_jobs(
-    openshift: OpenShift,
-    index_urls: List[str],
-    package_name: str,
-    package_version: str,
-    output: str,
+def _take_data_science_packages() -> List[str]:
+    """Take list of Python Packages for data science."""
+    current_path = Path.cwd()
+    complete_file_path = current_path.joinpath("hundredsDatasciencePackages.yaml")
+
+    with open(complete_file_path) as yaml_file:
+        requested_packages = yaml.safe_load(yaml_file)
+
+    data_science_packages = []
+    for package_name in requested_packages["hundreds_datascience_packages"]:
+        _LOGGER.info("Registering package %r", package_name)
+        data_science_packages.append(package_name)
+
+    return data_science_packages
+
+
+def _schedule_default_packages_solver_jobs(packages: List[str], index_urls: List[str]) -> None:
+    """Run solver jobs for Python packages list selected."""
+    openshift = OpenShift()
+
+    pypi = Source("https://pypi.org/simple")
+
+    for package_name in packages:
+        _LOGGER.debug(f"Obtainig {package_name} versions")
```

Comment:
```
        _LOGGER.debug("Obtainig %r versions", package_name)

```

Reply:
Ahhh btw, can we use ‚‘.format()? Any caveat?

Reply:
The string will be expanded lazily with:

```python
        _LOGGER.debug("Obtainig %r versions", package_name)
```

That means if the log level is higher than ``DEBUG`` no additional instructions will be produced. This saves some CPU cycles. I know it might be too micro-optimization.. but considering we log on different layers in the application with different log levels, it can in sum produce faster application and we might really feel the speed up.

Also, some linters will complain here.